src/xunit.performance.run/ProgramCore.cs

```diff
@@ -137,11 +141,25 @@ private void RunTests(XunitPerformanceProject project)
             };
 
             startInfo.Environment["XUNIT_PERFORMANCE_RUN_ID"] = project.RunId;
-            startInfo.Environment["XUNIT_PERFORMANCE_MIN_ITERATION"] = "10";
-            startInfo.Environment["XUNIT_PERFORMANCE_MAX_ITERATION"] = "1000";
-            startInfo.Environment["XUNIT_PERFORMANCE_MAX_TOTAL_MILLISECONDS"] = "1000";
-            startInfo.Environment["COMPLUS_gcConcurrent"] = "0";
-            startInfo.Environment["COMPLUS_gcServer"] = "0";
+
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("XUNIT_PERFORMANCE_MIN_ITERATION")))
+                startInfo.Environment["XUNIT_PERFORMANCE_MIN_ITERATION"] = MinIterations.ToString();
+
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("XUNIT_PERFORMANCE_MAX_ITERATION")))
+                startInfo.Environment["XUNIT_PERFORMANCE_MAX_ITERATION"] = MaxIterations.ToString();
+
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("XUNIT_PERFORMANCE_MAX_ITERATION_INNER_SPECIFIED")))
+                startInfo.Environment["XUNIT_PERFORMANCE_MAX_ITERATION_INNER_SPECIFIED"] = MaxIterationsWhenInnerIterationsSpecified.ToString();
+
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("XUNIT_PERFORMANCE_MAX_TOTAL_MILLISECONDS")))
+                startInfo.Environment["XUNIT_PERFORMANCE_MAX_TOTAL_MILLISECONDS"] = MaxTotalMSPerTest.ToString();
+
+            // Disable server and background GC.
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("COMPLUS_gcConcurrent")))
+                startInfo.Environment["COMPLUS_gcConcurrent"] = "0";
```

Comment:
The premise here is that the vast majority of tests are not GC tests, and should run under whatever GC mode produces the most consistent results.  GC tests are likely to need to run in their own process, with their own configuration, anyway.